packages/outputarea/src/widget.ts

```diff
@@ -936,15 +984,17 @@ export class Stdin extends Widget implements IStdin {
       node: Private.createInputWidgetNode(options.prompt, options.password)
     });
     this.addClass(STDIN_CLASS);
-    this._historyIndex = 0;
-    this._input = this.node.getElementsByTagName('input')[0];
-    this._trans = (options.translator ?? nullTranslator).load('jupyterlab');
-    // make users aware of the line history feature
-    this._input.placeholder = this._trans.__('↑↓ for history');
     this._future = options.future;
+    this._historyIndex = 0;
+    this._historyPat = '';
     this._parentHeader = options.parent_header;
-    this._value = options.prompt + ' ';
     this._password = options.password;
+    this._trans = (options.translator ?? nullTranslator).load('jupyterlab');
+    this._value = options.prompt + ' ';
+
+    this._input = this.node.getElementsByTagName('input')[0];
+    // make users aware of the line history feature
+    this._input.placeholder = this._trans.__(Stdin._placeholder);
```

Comment:
unfortunately this would not work - the translation extraction tooling requires the strings to be put within `this._trans.__()` (as it was before).

Reply:
didn't know that, kind of sucks. Done